### ISSUE TITLE
Commands for generate

### ISSUE THREAD

**Issue by credondocr** [kriasoft/react-static-boilerplate#104]
I think is important makes life easier, a generate component tool can be a awesome feature.

For example:

$ react-static-boilerplate component MyComponent

```shell
.
├── /components/                # Shared or generic UI components
│   └── /MyComponent   #New Component
│       └── /Files              # etc.
```

**Comment by koistya** [comment]
@credondocr yep, that will be a great feature to have, e.g. `node run new:component MyComponent` creates `/coponents/MyComponent`; `node run new:route /tasks/:id TaskDetails` creates `/pages/tasks/TaskDetails.js` or something like that.

**Comment by credondocr** [comment]
@koistya another way to implement that can be:
- yo react-static component MyComponent
- yo react-static route MyRoute
I really like Yeoman

**Comment by koistya** [comment]
Yep, I need to update `generator-react-static` with the latest version of RSB from master and these scaffolding tools.

**Comment by koistya** [comment]
@credondocr just scaffolded the bare minimum Yeoman generator here:

https://github.com/kriasoft/react-static-boilerplate/tree/generator-react-static

Next step is to add sub-generators for components and pages.
